applications/app/views/atomEmbed.scala.html

```diff
@@ -1,18 +1,31 @@
 @import model.AtomPage
-@import conf.Configuration
 
 @(page: AtomPage)(implicit request: RequestHeader, context: model.ApplicationContext)
 
 <!DOCTYPE html>
 <html lang="en-GB" class="gu-atom-embed-html">
-<head>
-    <meta charset="utf-8">
-    <title>@page.metadata.webTitle</title>
-    <base target="_top">
-    @fragments.atomPageHead(page)
-</head>
-<body>
-    @page.body
-    @fragments.atomPageFoot(page)
-</body>
+    <head>
+        <meta charset="utf-8">
+        <title>@page.metadata.webTitle</title>
+        <base target="_top">
+        @fragments.atomPageHead(page)
+    </head>
+    <body>
+        @page.body
+        @fragments.atomPageFoot(page)
+    </body>
+    <script>
+        // Interactive Atoms expect window.resize to exist. In the context of Amp we want to let the parent know that
+        // the iframe should be resized. See https://amp.dev/documentation/components/amp-iframe/.
+        function resize() {
```

Comment:
Is there possibly a race condition with the script that will fire this `window.resize` in the interactives vs this being at the bottom of the page? I.E, it could possibly maybe fire first however unlikely? Maybe this could go in the head.

Also, I wonder if it's worth explicitly specifying `window.resize = function() {...}` for clarity?

Reply:
Good questions! Yeah seems worth moving to the head just in case. I think in the interactive the call to `window.resize` is wrapped in a `setTimeout`, so it's probably unlikely, but yeah can't do any harm for sure. Definitely :+1: specifying `window.resize`.